### PR TITLE
add counter

### PR DESCRIPTION
**Pull request by gdams**
Adds a total counter to index.html

Screenshot 2019-03-14 at 17 19 51

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+const java_version = [
+  "openjdk8",
+  "openjdk9",
+  "openjdk10",
+  "openjdk11"
```
Comment (karianna) [inline index.js:5]: Should we pre add 12?  Or can we ignore the version number even?  Else if we forge to update this.

File: style.css [hunk 2]
```diff
@@ -7,3 +7,36 @@ canvas#graph {
...
   height: 60% !important;
 }
+
+.col {
+  padding-top: 1rem;
+}
+
+.counter {
+  background-color: #f5f5f5;
+  padding: 20px 0;
```
Comment (karianna) [inline style.css]: use em's not px

File: index.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    "url": `https://api.adoptopenjdk.net/v2/info/releases/${version}`,
+    "method": "GET"
+  }
+
+  $.ajax(settings).done(function(releases) {
+    counter += 1;
+    for (var release of releases) {
+      total += release.download_count
+    }
+    if (counter == java_version.length) {
```
Comment (karianna) [inline index.js:30]: OK, can we split out some of the logic into named functions?  This seems to be the classic for if for nested JS of doom :-)

**Review by karianna — CHANGES_REQUESTED** [review]
Some structural changes needed I think

File: index.js (hunk 1 above)
Comment (gdams) [reply index.js:5]: Can't add jdk12 yet. It returns a 404 atm as there are no releases.

File: style.css (hunk 2 above)
Comment (gdams) [reply style.css]: done

File: index.js (hunk 3 above)
Comment (gdams) [reply index.js:30]: done

File: index.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    function formatter(value, settings) {
+      return value.toFixed(settings.decimals);
+    }
+  }(jQuery));
+
+  jQuery(function ($) {
+    // custom formatting example
+    $('.count-number').data('countToOptions', {
+    formatter: function (value, options) {
+      return value.toFixed(options.decimals).replace(/\B(?=(?:\d{3})+(?!\d))/g, ',');
```
Comment (karianna) [inline index.js:123]: Sample of what that looks like?

File: index.js [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    }
+    });
+
+    // start all the timers
+    $('.timer').each(count);
+
+    function count(options) {
+    var $this = $(this);
```
Comment (karianna) [inline index.js]: white space

**Review by karianna — CHANGES_REQUESTED** [review]
couple more nitpicks then this is good

File: index.js (hunk 4 above)
Comment (gdams) [reply index.js:123]: this adds commas to each 3 numbers: `15125946` is formatted as `15,125,946`

**Review by karianna — APPROVED** [review]

**Pull request merged**